src/components/ShowList.jsx

```diff
@@ -31,11 +43,38 @@ function ShowList() {
       )}
       {value && value.docs.length > 0 && (
         <div>
-          Collection:
+          <input
+            type="text"
+            placeholder="Filter items..."
+            value={filter}
+            onChange={handleChange}
+          />
+          {filter && (
+            <button
+              onClick={() => setFilter('')}
+              aria-label="clear filter text"
+            >
+              <FontAwesomeIcon icon={['fas', 'times']} />
```

Comment:
Hi both, this all looks great. I was wondering what the function of` 'times' `is here.

Reply:
Hi @blackbureau, That is the name of our icon from fontawesome

Reply:
Thanks @ishimwezachee 👍🏾